package.json

```diff
@@ -57,6 +57,7 @@
     "passport-jwt": "^4.0.0",
     "pg": "^8.7.1",
     "pg-escape": "^0.2.0",
+    "pluralize": "^8.0.0",
```

Comment:
I think this might be tech debt to think about.. some time in the future. Generally depts should have a "welsh language scheme" detailing the department's support of the welsh language (I can't find it very easily..) so an i18n lib would be better suited if FCDO would, one day, need to supply this in Welsh. Anyway - not a big deal for now. 


addendum, I've just asked, and FCDO doesn't have one, and might not even need one.. so this might have been ramble for nothing..